api/apiclient_test.go

```diff
@@ -1125,6 +1240,10 @@ type loginTimeoutAPIAdmin struct {
 }
 
 func (a *loginTimeoutAPIAdmin) Login(req params.LoginRequest) (params.LoginResult, error) {
-	<-a.r.unblock
+	ch, ok := <-a.r.unblock
```

Comment:
It's in a test, which controls both the sender and the receiver. IMO this is fine.